MapboxNavigation/NavigationMapView.swift

```diff
@@ -60,6 +60,29 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      */
     public weak var courseTrackingDelegate: NavigationMapViewCourseTrackingDelegate?
     
+    /**
+     Controls whether delays are shown along the route line.
+     */
+    public var showsDelay: Bool = false {
```

Comment:
For now boolean flag is used to show/hide delay information views. Since delay information is provided for `Route`, `Leg` and `Step` we might want to provide API which allows to show delay for each of those. As per comment [here](https://github.com/mapbox/navigation-sdks/issues/373#issuecomment-672027950) default minimum dealy threshold value should be set to `4 minutes`.